src/App.js

```diff
@@ -42,10 +45,10 @@ export default class App extends React.Component {
       <>
         <TopBar />
         <main className="container">
-          <section className="sidebar">{/* Recent Donations */}</section>
+          <section className="sidebar"><RecentDonations /></section>
```

Comment:
Ok James, in regards to the third test, logging out the donations. First, you're not passing any props to the RecentDonations component here. Remember, props are passed as attributes to a JSX element, just like you did to Progress below with donations = {donations}.

Reply:
copy! todays review helped me understand it a bit more so I'll definitely fix that